quizzes/09_arrow.test.js

```diff
@@ -0,0 +1,77 @@
+console.log(
+  // curryAdd(2)(4)
+  // genSeries(7, 100)
+  // shift("havin pasta for brunch")
+)
+
+// population.display()
+
+function curryAdd(a){
+  // TODO: refactor this function
+  // rewrite with arrow syntax
+  // remove function and return keywords
+  return function(b) {
+    return a + b;
+  }
+}
+
+function genSeries(n=10, limit=10) {
+  // TODO: refactor this function
+  // remove all keywords like function and return
+  // remove the loop
+  // Use spread opertaor, Array functions in ES6, like Array.from()
+
+  // this function returns an array of n random integers
+  // Each integer can be less than or equal to limit
+  // limit is 10 by default, if not provided
+  let arr = [];
+
+  for(let i = 0 ; i < n ; i++) {
+    arr.push(Math.floor(Math.random() * limit))
+  }
+  return arr;
+}
+
+/**
+ * This is a complex example, be careful while refactoring.
+ */
+function shift(str, amount=1) {
+  // TODO: shift() shifts the input string
+  // assume str is made of only lowercase alphabet letters
+  // update the RegEx below
+  str.replace(/\/g, function(x){
```

Comment:
I think that we need to `return` here :) Also, what's your expected implementation here. 😅 I'm not sure I know :)